0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/Command.scala

```diff
@@ -264,10 +270,17 @@ object Command {
           action { (_, c) => c.copy(skipWarnings = true) }
           text "Don't output messages with log level less than ERROR",
 
-        opt[SeverityLevel]("severityLevel")
-          action { (x, c) => c.copy(severityLevel = x) }
-          text "Severity level\t\t\t\tDefault: 1"
-
+        opt[List[Linter]]("skip-checks")
+          action { (x, c) => c.copy(linters = x) }
+          valueName "<linters>"
+          text "Lint without specified linters, given comma separated\tDefault: None\n" +
+                "\t\t\t   All linters and their explanations are below.\n" +
```

Comment:
This should have been derived from `allowedToBeExcluded` or other place where we have names.